### DISCUSSION TITLE
Some globally installed packages aren't vailable to LazyVim

### DISCUSSION THREAD

**Discussion by Pilgrim1379** [LazyVim/LazyVim#3668] [Q&A]
When I run `LazyHealth` I get a few warnings about packages not being available. However this packages are indeed installed and I can access them via the command line. Below are screenshots of the sections with the warnings.

<img width="897" alt="Screenshot 2024-06-15 at 11 34 18" src="https://github.com/LazyVim/LazyVim/assets/219075/5bffee61-8425-488e-ab68-b3c668e12810">

So for the above `elm-format, prettier and zig fmt` are all installed.

<img width="897" alt="Screenshot 2024-06-15 at 11 35 04" src="https://github.com/LazyVim/LazyVim/assets/219075/0e51b0f3-ef18-4dd2-ad7c-97cdf539637b">

With this, I don't have any private `lspconfig.hls` only the `haskell extra` installed.

<img width="870" alt="Screenshot 2024-06-15 at 11 35 51" src="https://github.com/LazyVim/LazyVim/assets/219075/20770ad2-8ce1-4e2d-be7b-527bd90a6c18">

And finally, I don't have `yarn` installed as I don't use it. I prefer `bun` instead `npm` instead.

Hoping for some explanations or answers. Thanks

**Comment by dpetka2001** [comment]
If your dotfiles are up to date, then you set up `hls` [here](https://github.com/Pilgrim1379/dotfiles/blob/3c9a7d071e3a9e6d0146651297281d39480d9596/private_dot_config/LazyVim/lua/plugins/lspconfig.lua#L55-L58). 

For `prettier` try to update to latest LazyVim version and I believe you shouldn't see this problem. 

About `zig` and `elm` you have misconfigured `conform.nvim`. Read the docs in the repo for how to configure them (specifically [this section](https://github.com/stevearc/conform.nvim?tab=readme-ov-file#customizing-formatters)), as I personally don't use them.
